### ISSUE TITLE
CSV loader

### ISSUE THREAD

**Issue by nkint** [accurat/accurapp#18]
Now done with:

`yarn add -D csv-loader`

and in `webpack.config.js`:

```
const buildWebpackConfig = require('webpack-preset-accurapp')
module.exports = buildWebpackConfig([
  (context) => ({
    module: {
      loaders: [
        {
          test: /\.csv$/,
          loader: 'csv-loader',
          options: {
            dynamicTyping: true,
            header: true,
            skipEmptyLines: true,
          },
        },
      ],
    },
  }),
])
```

**Comment by markov00** [comment]
I don't like so much the idea behind the csv loader. If it's a relatively small file it is fine, but with large CSV,  you will end up having a big bundle.js file with the JSONified CSV in it.

**Comment by marcofugaro** [comment]
How about importing the csv with the `raw-loader` and parsing it with papaparse or d3 or something else?

Concerning the big csvs issue it's on the developer to be careful with that.

cc @caesarsol

**Comment by caesarsol** [comment]
It's more useful in the prototyping phase than in production, so the "bundle too big" problem can be forgot.

It's very useful for loght data analysis, for example for @gio_magni

**Comment by marcofugaro** [comment]
Closed in #40
